docs/operating/manuals/resourceconsist.md

```diff
@@ -46,28 +50,40 @@ type ReconcileWatchOptions interface {
 	EmployeePredicates() predicate.Funcs
 }
 
+
+// ReconcileLifecycleOptions defines whether PodOpsLifecycle followed and
+// whether employees' LifecycleFinalizer conditions need to be Record/Erase to employer's anno.
```

Comment:
change to `need to be Recorded to or erased from employer's anno`?

Reply:
changed